PolyEngine/Core/Src/String.hpp

```diff
@@ -48,7 +48,7 @@ namespace Poly {
 		}
 
 		//------------------------------------------------------------------------------
-		bool operator==(const T* str) {
+		bool operator==(const T* str) const {
```

Comment:
I'm not sure about this, but without this operators beeing const I could not compare Path.Get() with other Strings. So I made them const just like they are for Matrix class.